src/runtime/cuda.cpp

```diff
@@ -576,13 +591,15 @@ namespace Internal {
 WEAK halide_device_allocation_pool cuda_allocation_pool;
 
 __attribute__((constructor))
-WEAK void register_cuda_allocation_pool() {
+WEAK void
+register_cuda_allocation_pool() {
```

Comment:
I see this in a few places. Not sure what's going on.

Reply:
Moving the __attribute__ part to the end of the function make clang-format happier. (AFAICT it's legal in either position; correct me if I'm wrong.)